test/binary/bad-export-out-of-range.txt

```diff
@@ -0,0 +1,11 @@
+;;; TOOL: run-gen-wasm-bad
+magic
+version
+section(TYPE) { count[1] function params[0] results[0] }
+section(FUNCTION) { count[1] sig[0] }
+section(EXPORT) { count[1] str("foo") func_kind func[0xff] func[0xff] func[0xff] func[0xff] func[0x0f] }
```

Comment:
you can change this to `func[leb_u32(4294967295)]` to make the test a bit clearer